Resources/Prototypes/explosion.yml

```diff
@@ -19,6 +19,7 @@
   lightColor: Orange
   texturePath: /Textures/Effects/fire.rsi
   fireStates: 3
+  fireStacks: 1
```

Comment:
This is a bad approach, any explosion will add firestacks to flammable entities, e.g. Dionas and Moths. I don't think that's desirable.

Reply:
Indeed.  I didn't talk about that.  I'll expose a temperature like you suggested